packages/components/src/summary/number.js

```diff
@@ -60,6 +61,7 @@ const SummaryNumber = ( {
 	if ( ! onToggle ) {
```

Comment:
Just for my clarification, this is called only when `onToggle` is passed from a dropdown when this is a button, right?

Optional, but what do you think about creating a constant `isButton` instead of checking `onToggle`?  Took me a second to figure out what was going on here and this might make it a tiny bit faster to understand.